.github/workflows/build-without-push-docker.yml

```diff
@@ -0,0 +1,42 @@
+name: Docker Build without Push
+
+on:
+  pull_request:
+    branches: [main]
```

Comment:
I would have thought we don't want to run this one on `main`?

Reply:
Doesn't this mean - on "pull request to main"? And we wouldn't be creating a PR from main to main.